docs/about-filecoin/what-is-filecoin.md

```diff
@@ -5,19 +5,19 @@ description: Explore Filecoin, a peer-to-peer network that stores files, with bu
 
 # What is Filecoin?
 
-**Filecoin is a peer-to-peer network that stores files, with built-in economic incentives to ensure files are stored reliably over time.**
+Filecoin is a peer-to-peer network that stores files, with built-in economic incentives to ensure files are stored reliably over time.
 
-In Filecoin, users pay to store their files on storage miners. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage and pricing is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
+In Filecoin, users pay to store their files on storage miners. Storage miners are computers responsible for storing files and proving they have stored the files correctly over time. Anyone who wants to store their files or get paid for storing other users’ files can join Filecoin. Available storage, and the price of that storage, is not controlled by any single company. Instead, Filecoin facilitates open markets for storing and retrieving files that anyone can participate in.
 
 Filecoin includes a blockchain and native cryptocurrency (FIL). Storage miners earn units of FIL for storing files. Filecoin’s blockchain records transactions to send and receive FIL, along with proofs from storage miners that they are storing their files correctly.
 
-![Diagram showing a four-step process for storing files with Filecoin. First step: User who wants to store a file pays miner to store their file. Second step: Miner commits publicly via Filecoin’s blockchain to storing the file. Third step: The network constantly verifies that miners are storing files correctly. Fourth step: User pays miner to retrieve their file.](./images/what-is-filecoin/what-is-filecoin-diagram.png)
+![The four-step process for storing files with Filecoin.](./images/what-is-filecoin/what-is-filecoin-diagram.png)
```

Comment:
Whoever wrote this image description was probably thinking of describing the image for visually impared users that cannot see it. I don't see anything wrong with that.

Reply:
(reverted)